rules/sairedis.dep

```diff
@@ -0,0 +1,17 @@
+
+#DPKG FRK
+SPATH       := $($(LIBSAIREDIS)_SRC_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/sairedis.mk rules/sairedis.dep   
+DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+
+SMDEP_PATHS := $(SPATH) $(SPATH)/SAI $(SPATH)/SAI/bm/behavioral-model $(SPATH)/SAI/test/ptf $(SPATH)/SAI/test/saithrift/ctypesgen
+$(foreach path, $(SMDEP_PATHS), $(eval $(path) :=$(filter-out $(SMDEP_PATHS),$(addprefix $(path)/, \
+					$(shell cd $(path)  && git ls-files | grep -v " ")))))
```

Comment:
Build error as below, it may be caused by the space in file name.
sonic-buildimage/src/sonic-sairedis$ ls SAI/doc/OCP/
'**SAI** Demo 2016.pptx'  '**SAI** Demo 2017 topology.pdf'

12:10:19  make: *** No rule to make target 'src/sonic-sairedis/SAI/doc/OCP/SAI', needed by 'target/debs/stretch/libsaithrift-dev_0.9.4_amd64.deb.smdep'.  Stop.
12:10:43  Makefile.work:209: recipe for target 'target/docker-ptf-brcm.gz' failed
12:10:43  make[1]: *** [target/docker-ptf-brcm.gz] Error 2